ui/explore/src/components/PrometheusMetricsFinder/filter/FilterInputs.tsx

```diff
@@ -215,7 +215,7 @@ export function RawFilterInput({
             />
           );
         }}
-        onChange={(_: SyntheticEvent, newValue: string[] | string | null) => {
+        onChange={(_, newValue) => {
```

Comment:
why did you need to remove every type here ? I thought the previous PRs you made was to add them

Reply:
after talking with Guillaume, it seems for obscure reason, we cannot keep the type for this particular one as it doesn't compile afterward.

So let's drop it